### PR TITLE
#159273994 Implement Learning Facilitation Assistant Feedback

### PR DESCRIPTION
**Pull request by chinemelu**
#### What does this PR do?
Implement feedback from LFA

#### Description of Task to be completed?
* Adjust avatar section in the profile page
* Improve the look of the navigation bar with smaller screens
* Prevent API from 'breaking' when undefined fields are submitted
* Modify PUT /api/v1/entries route to allow modification with undefined input fields

#### What are the relevant pivotal tracker stories?
#159273994

### CODE REVIEW

**Comment by chinemelu** [comment]
@torkpe please review my pull request

File: server/validators/postDiaryEntryValidator/titleValidator.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
 import validator from 'validator';
 
-const postDiaryEntryTitleValidator = (data) => {
+const postDiaryEntryTitleValidator = (data, isEditing, diaryEntry) => {
```
Comment (torkpe) [inline server/validators/postDiaryEntryValidator/titleValidator.js:3]: Nice work you did here. Making adding an additional parameter to check if it's a `PUT` or `POST` request

**Review by torkpe — COMMENTED** [review]
Nice. LGTM

File: server/validators/postDiaryEntryValidator/titleValidator.js (hunk 1 above)
Comment (chinemelu) [reply server/validators/postDiaryEntryValidator/titleValidator.js:3]: Thank you @torkpe for your feedback and help.

**Pull request merged**